build.gradle

```diff
@@ -10,8 +10,8 @@ repositories {
 	maven { url "https://jitpack.io" }
 }
 
-sourceCompatibility = JavaVersion.VERSION_17
-targetCompatibility = JavaVersion.VERSION_17
+sourceCompatibility = JavaVersion.VERSION_11
+targetCompatibility = JavaVersion.VERSION_11
```

Comment:
Please revert this to make the diff cleaner, as this change is covered by your PR #42.